Resfeber/Helpers/Notifications.swift

```diff
@@ -12,4 +12,5 @@ extension Notification.Name {
     static let oktaAuthenticationSuccessful = Notification.Name("oktaAuthenticationSuccessful")
     static let oktaAuthenticationFailed = Notification.Name("oktaAuthenticationFailed")
     static let oktaAuthenticationExpired = Notification.Name("oktaAuthenticationExpired")
+    static let loadData = Notification.Name("loadData")
```

Comment:
I think we should update the notification name to something more descriptive of the event that triggers the notification. Suggestion:
```suggestion
    static let tripDataUpdated = Notification.Name("tripDataUpdated")
```